src/core/ledger/ledger.js

```diff
@@ -744,10 +763,15 @@ export class Ledger {
 
   // Helper method for recording that Grain was allocated to a identity.
   // Increases the identity's paid amount and balance in sync.
-  _allocateGrain(recipient: IdentityId, amount: G.Grain) {
-    const account = this._mutableAccount(recipient);
-    account.paid = G.add(amount, account.paid);
-    account.balance = G.add(amount, account.balance);
+  _allocateGrain(
+    grainReceipt: GrainReceipt,
+    allocationId: AllocationId,
+    timestampMs: TimestampMs
```

Comment:
nit / for readability, a weak preference that we refactor this to take an object with named arguments rather than positional args.

Reply:
accepted